src/components/20-molecules/datepicker/story.js

```diff
@@ -14,9 +14,12 @@ import Changelog from './CHANGELOG.md';
 
 const localeOptions = {
   'de-CH': 'de-CH',
+  'it-CH': 'it-CH',
+  'fr-CH': 'fr-CH',
+  'en-CH': 'en-CH',
   'en-GB': 'en-GB',
   'it-IT': 'it-IT',
-  'fr-CH': 'fr-CH',
+  'invalid/ unsupported': 'ff-XX',
```

Comment:
What is the use of this? If this is not something trivial i would recommend to write a comment.

Reply:
To show the default language fallback scenario. Will add a comment